source/api_documentation/network_integration/promo_numbers/index.rst

```diff
@@ -134,7 +134,23 @@ When you POST a promo number, we return the promo_number in the response body. T
     - string (optional)
     - The ID of the AdWords client account to be associated with this promo number. Requires media type to be set to 'Google Call Extension'. (ex. "627-345-9815")
 
+  * - destination_advertiser_id_from_network
+    - string (optional)
+    - The ID of the advertiser in which the promo number is intended to be moved to.  If this field is used, **destination_advertiser_campaign_id_from_network** also must be included.
+
+      * Do not include this field if you do not intend to move the promo number.
```

Comment:
❓ Is adding the bullet point for each of these intentional?

<img width="1357" alt="image" src="https://github.com/Invoca/developer-docs/assets/9993068/a5e4abec-9cd5-4416-9dea-79952238f405">

Reply:
I think this just be a difference in how GitHub renders the markdown-like syntax we use in the docs.  I just did a quick sanity check and I'm still seeing what's linked in the Updated section of the PR's description.